### ISSUE TITLE
Latest master does not work

### ISSUE THREAD

**Issue by raine** [idleloop-github/xbmc-earthcam#4]
```
2019-09-26 21:38:20.859 T:3011522576   ERROR: CCurlFile::Stat - Failed: HTTP response code said error(22) for https://videos3.earthcam.com/fecnetwork/tokyo1.flv/chunklist_w1622545460.m3u8
2019-09-26 21:38:20.902 T:3011522576  NOTICE: VideoPlayer::OpenFile: https://videos3.earthcam.com/fecnetwork/tokyo1.flv/chunklist_w1622545460.m3u8
2019-09-26 21:38:21.894 T:3011522576   ERROR: CCurlFile::Stat - Failed: HTTP response code said error(22) for https://videos3.earthcam.com/fecnetwork/tokyo1.flv/chunklist_w1622545460.m3u8
2019-09-26 21:38:21.898 T:2390750064  NOTICE: Creating InputStream
2019-09-26 21:38:22.092 T:2452706160   ERROR: GetDirectory - Error getting plugin://plugin.video.earthcam/?Serie&action=play&category&channel&extradata&fulltitle=Tokyo%20Cam&plot=Tokyo%20Cam%0a(Tokyo%2c%20Jap an)&show&thumbnail=https%3a%2f%2fstatic.earthcam.com%2fcamshots%2f512x288%2fe6f806baf57db637303a78b77f7af4f4.jpg&title=Tokyo%20Cam&url=https%3a%2f%2fwww.earthcam.com%2fworld%2fjapan%2ftokyo%2f
2019-09-26 21:38:22.122 T:3011522576   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.earthcam/?Serie&action=play&category&channel&extradata&fulltitle=Tokyo%20Cam&plot=Tokyo%20Cam%0a(Tokyo%2c%20Ja pan)&show&thumbnail=https%3a%2f%2fstatic.earthcam.com%2fcamshots%2f512x288%2fe6f806baf57db637303a78b77f7af4f4.jpg&title=Tokyo%20Cam&url=https%3a%2f%2fwww.earthcam.com%2fworld%2fjapan%2ftokyo%2f) failed
2019-09-26 21:38:22.624 T:2390750064  NOTICE: Creating Demuxer
2019-09-26 21:38:23.243 T:2390750064   ERROR: OpenDemuxStream - Error creating demuxer
2019-09-26 21:38:23.243 T:2390750064  NOTICE: CVideoPlayer::OnExit()
2019-09-26 21:38:23.256 T:3011522576  NOTICE: CVideoPlayer::CloseFile()
2019-09-26 21:38:23.256 T:3011522576  NOTICE: VideoPlayer: waiting for threads to exit
2019-09-26 21:38:23.256 T:3011522576  NOTICE: VideoPlayer: finished waiting
```

I'm running Kodi 18.4.

**Comment by idleloop-github** [comment]
Hi @raine !
I cannot actually reproduce your problem:
* Are you using this addon? Go to information and check version: this is v1.3.2. Please, note that there's another addon available from Kodi repositories with this same name: see https://github.com/idleloop-github/xbmc-earthcam/issues/2#issuecomment-375890743. My addon [must be installed manually](https://github.com/idleloop-github/xbmc-earthcam#installation) :-)
* Sometimes a particular camera may not be available one day but the rest are available as usual
* You can change from HLS to RTMP in Configuration of the plugin to see if this change something (or vice versa)
* Check [this notes](https://github.com/idleloop-github/xbmc-earthcam#note-for-kodi-17-krypton) to see if something applies to you (in case you have a specific HW like RPi, Android, etc)

**Comment by raine** [comment]
Yes, I installed master of this repo. Also, I tried multiple streams but none worked. 

Let me try again later after changing the settings.

**Comment by raine** [comment]
Nope. This is what I see after trying to play some streams from the featured list:

```
2019-10-01 19:39:56.040 T:3011859040  NOTICE: VideoPlayer::OpenFile: rtmp://videos3.earthcam.com/ app=fecnetwork swfUrl=https://www.earthcam.com/swf/cam_player_v2/ecnPlayer.swf playpath=15989.flv live=true ti
meout=10
2019-10-01 19:39:56.049 T:2324771696  NOTICE: Creating InputStream
2019-10-01 19:39:56.050 T:2324771696  NOTICE: Creating Demuxer
2019-10-01 19:39:56.207 T:3011855216   ERROR: CCurlFile::FillBuffer - Failed: Couldn't connect to server(7)
2019-10-01 19:39:56.207 T:3011855216   ERROR: CCurlFile::Open failed with code 0 for http://192.168.1.199:51226/ctrl-int/1/play:
2019-10-01 19:39:56.208 T:3011855216   ERROR: Open - failed to open source <http://192.168.1.199:51226/ctrl-int/1/play|Active-Remote=1584197151>
2019-10-01 19:39:56.216 T:2510807920   ERROR: GetDirectory - Error getting plugin://plugin.video.earthcam/?Serie&action=play&category&channel&extradata&fulltitle=Downtown%20Franklin%20Cam&plot=Downtown%20Fran
klin%20Cam%0a(Franklin)&show&thumbnail=https%3a%2f%2fstatic.earthcam.com%2fcamshots%2f512x288%2fc8d9c66b3292cd6a6279b0f4635afeb7.jpg&title=Downtown%20Franklin%20Cam&url=http%3a%2f%2fwww.earthcam.com%2fusa%2fp
ennsylvania%2ffranklin%2f
2019-10-01 19:39:56.226 T:3011859040   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.earthcam/?Serie&action=play&category&channel&extradata&fulltitle=Downtown%20Franklin%20Cam&plot=Downtown%20Fra
nklin%20Cam%0a(Franklin)&show&thumbnail=https%3a%2f%2fstatic.earthcam.com%2fcamshots%2f512x288%2fc8d9c66b3292cd6a6279b0f4635afeb7.jpg&title=Downtown%20Franklin%20Cam&url=http%3a%2f%2fwww.earthcam.com%2fusa%2f
pennsylvania%2ffranklin%2f) failed
2019-10-01 19:39:56.498 T:2324771696   ERROR: OpenDemuxStream - Error creating demuxer
2019-10-01 19:39:56.498 T:2324771696  NOTICE: CVideoPlayer::OnExit()
2019-10-01 19:39:56.525 T:3011859040  NOTICE: CVideoPlayer::CloseFile()
2019-10-01 19:39:56.526 T:3011859040  NOTICE: VideoPlayer: waiting for threads to exit
2019-10-01 19:39:56.526 T:3011859040  NOTICE: VideoPlayer: finished waiting
2019-10-01 19:39:56.618 T:3011855216   ERROR: CCurlFile::FillBuffer - Failed: Couldn't connect to server(7)
2019-10-01 19:39:56.618 T:3011855216   ERROR: CCurlFile::Open failed with code 0 for http://192.168.1.199:51226/ctrl-int/1/stop:
2019-10-01 19:39:56.618 T:3011855216   ERROR: Open - failed to open source <http://192.168.1.199:51226/ctrl-int/1/stop|Active-Remote=1584197151>
2019-10-01 19:40:01.639 T:3011859040  NOTICE: VideoPlayer::OpenFile: rtmp://videos3.earthcam.com/ app=fecnetwork swfUrl=https://www.earthcam.com/swf/cam_player_v2/ecnPlayer.swf playpath=9974.flv live=true tim
eout=10
2019-10-01 19:40:01.646 T:2423255920  NOTICE: Creating InputStream
2019-10-01 19:40:01.647 T:2423255920  NOTICE: Creating Demuxer
2019-10-01 19:40:01.690 T:3011855216   ERROR: CCurlFile::FillBuffer - Failed: Couldn't connect to server(7)
2019-10-01 19:40:01.690 T:3011855216   ERROR: CCurlFile::Open failed with code 0 for http://192.168.1.199:51226/ctrl-int/1/play:
2019-10-01 19:40:01.690 T:3011855216   ERROR: Open - failed to open source <http://192.168.1.199:51226/ctrl-int/1/play|Active-Remote=1584197151>
2019-10-01 19:40:01.774 T:2519462768   ERROR: GetDirectory - Error getting plugin://plugin.video.earthcam/?Serie&action=play&category&channel&extradata&fulltitle=Times%20Square%20Street%20Cam&plot=Experience%
20this%20street-level%20view%20looking%20southeast%20toward%20the%20heart%20of%20Times%20Square.%20Perched%2012%20feet%20above%20the%20sidewalk%20on%20Broadway%20between%2046th%20and%2047th%20Streets%2c%20wat
ch%20the%20crowds%20as%20they%20stop%20in%20front%20of%20this%20live%20webcam%20to%20wave%20home%20to%20family%20and%20friends.&show&thumbnail=https%3a%2f%2fstatic.earthcam.com%2fcamshots%2f256x144%2f144ede00
2477031a0cf7d7635ee1a744.jpg&title=Times%20Square%20Street%20Cam&url=rtmp%3a%2f%2fvideos3.earthcam.com%2f%20app%3dfecnetwork%20swfUrl%3dhttps%3a%2f%2fwww.earthcam.com%2fswf%2fcam_player_v2%2fecnPlayer.swf%20p
laypath%3d9974.flv%20live%3dtrue%20timeout%3d10
2019-10-01 19:40:01.793 T:3011859040   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.earthcam/?Serie&action=play&category&channel&extradata&fulltitle=Times%20Square%20Street%20Cam&plot=Experience
%20this%20street-level%20view%20looking%20southeast%20toward%20the%20heart%20of%20Times%20Square.%20Perched%2012%20feet%20above%20the%20sidewalk%20on%20Broadway%20between%2046th%20and%2047th%20Streets%2c%20wa
tch%20the%20crowds%20as%20they%20stop%20in%20front%20of%20this%20live%20webcam%20to%20wave%20home%20to%20family%20and%20friends.&show&thumbnail=https%3a%2f%2fstatic.earthcam.com%2fcamshots%2f256x144%2f144ede0
02477031a0cf7d7635ee1a744.jpg&title=Times%20Square%20Street%20Cam&url=rtmp%3a%2f%2fvideos3.earthcam.com%2f%20app%3dfecnetwork%20swfUrl%3dhttps%3a%2f%2fwww.earthcam.com%2fswf%2fcam_player_v2%2fecnPlayer.swf%20
playpath%3d9974.flv%20live%3dtrue%20timeout%3d10) failed
2019-10-01 19:40:02.122 T:2423255920   ERROR: OpenDemuxStream - Error creating demuxer
2019-10-01 19:40:02.122 T:2423255920  NOTICE: CVideoPlayer::OnExit()
2019-10-01 19:40:02.146 T:3011855216   ERROR: CCurlFile::FillBuffer - Failed: Couldn't connect to server(7)
2019-10-01 19:40:02.146 T:3011855216   ERROR: CCurlFile::Open failed with code 0 for http://192.168.1.199:51226/ctrl-int/1/stop:
2019-10-01 19:40:02.146 T:3011855216   ERROR: Open - failed to open source <http://192.168.1.199:51226/ctrl-int/1/stop|Active-Remote=1584197151>
2019-10-01 19:40:02.159 T:3011859040  NOTICE: CVideoPlayer::CloseFile()
2019-10-01 19:40:02.159 T:3011859040  NOTICE: VideoPlayer: waiting for threads to exit
2019-10-01 19:40:02.159 T:3011859040  NOTICE: VideoPlayer: finished waiting
```

I'm not exactly sure what the calls to `http://192.168.1.199:51226` are. They seem to be related AirPlay, but .199 is my laptop's IP.

**Comment by idleloop-github** [comment]
Uhm... it seems like something is doing a MITM there... I'd bet for an antivirus or something like that... seems possible? doesn't seem to be related with this addon in particular... other addons run well?

**Comment by raine** [comment]
No issues with other addons. This is the only one I've had problems with.

**Comment by idleloop-github** [comment]
Can you turn on "debug" on this addon's settings to see if there's some more information dumped to logs?
I do not know about AirPlay... maybe there's some bad interaction there... Could you turn it off somehow momentarily to discard this?
